tajo-pullserver/src/main/java/org/apache/tajo/pullserver/FadvisedFileRegion.java

```diff
@@ -158,9 +160,9 @@ public void releaseExternalResources() {
    * we don't need the region to be cached anymore.
    */
   public void transferSuccessful() {
-    if (PullServerUtil.isNativeIOPossible() && manageOsCache && getCount() > 0) {
+    if (PullServerUtil.isNativeIOPossible() && manageOsCache && count() > 0) {
```

Comment:
Interesting... Netty team added isOpen() api on DefaultFileRegion class in 4.0.25.final. It could be good to use netty api for checking if this fileregion is deallocated or not.